src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -63,32 +63,90 @@ public ConfigurationClient(string connectionString, ConfigurationClientOptions o
         /// </summary>
         /// <param name="setting"><see cref="ConfigurationSetting"/> to create.</param>
         /// <param name="cancellation">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        public virtual async Task<Response<ConfigurationSetting>> AddAsync(ConfigurationSetting setting, CancellationToken cancellation = default)
+        public virtual async Task<Response<ConfigurationSetting>> AddAsync(string key, string value, string label = default, CancellationToken cancellationToken = default)
         {
-            if (setting == null) throw new ArgumentNullException(nameof(setting));
-            if (string.IsNullOrEmpty(setting.Key)) throw new ArgumentNullException($"{nameof(setting)}.{nameof(setting.Key)}");
+            if (string.IsNullOrEmpty(key)) throw new ArgumentNullException($"{nameof(key)}");
+            return await AddAsync(new ConfigurationSetting(key, value, label), cancellationToken);
+        }
+
+        public virtual Response<ConfigurationSetting> Add(string key, string value, string label = default, CancellationToken cancellationToken = default)
```

Comment:
Now that we have documentation per each public method, could you please add it to the sync ones?

Reply:
Good point, missed that.

Reply:
Ugh, merge really messed up doc comments. I'll fix everything up.